apps/aecore/src/aec_consensus_hc.erl

```diff
@@ -380,14 +384,15 @@ seal_correct_signature(Header, Signature, _Padding) ->
             {error, signature_verification_failed}
     end.
 
-generate_key_header_seal(_, Candidate, _PCHeight, #{expected_key_block_rate := _Expected} = _Config, _) ->
+generate_key_header_seal(_, Candidate, _PCHeight, #{child_block_time := ChildBlockTime} = _Config, _) ->
     Leader = aec_headers:beneficiary(Candidate),
     SignModule = get_sign_module(),
     case SignModule:set_candidate(Leader) of
         {error, key_not_found} ->
             timer:sleep(1000),
             {continue_mining, {error, no_solution} };
         ok ->
+            timer:sleep(ChildBlockTime),
```

Comment:
What is the purpose of sleeping in the code for signing a key block header? (The sleep in the error branch also looks dubious, but that seems legacy...)

Reply:
I assume this logic will move elsewhere once we start having proper time-slots etc, so perhaps this can stay for now.